File: dbflow-tests/src/androidTest/java/com/raizlabs/android/dbflow/test/myDB.java
package com.raizlabs.android.dbflow.test;

import com.raizlabs.android.dbflow.annotation.Database;

/**
 * Description:
 */
@Database(name = myDB.NAME, version = myDB.VERSION)
public class myDB {
    public static final String NAME = "test";
    public static final int VERSION = 1;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/myDB.kt
package com.raizlabs.android.dbflow.test

import com.raizlabs.android.dbflow.annotation.Database

/**
 * Description:
 */
@Database(name = myDB.NAME, version = myDB.VERSION)
object myDB {
    const val NAME = "test"
    const val VERSION = 1
}
